### PR TITLE
NO_TICKET - fix crash when opening feature toggle screen

### PR DESCRIPTION
**Pull request by gslevinkov**
+ fix group in firebase app distrub

## Jira Ticket

NO_TICKET

## Description of Work

-  fix crash when opening feature toggle screen

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/debug/featuretoggles/FeatureTogglesFragment.kt [hunk 1]
```diff
@@ -18,29 +18,27 @@ class FeatureTogglesFragment :
...
 
     companion object {
-        fun create() = FeatureTogglesFragment()
+        fun create(): FeatureTogglesFragment = FeatureTogglesFragment()
     }
 
     override val presenter: FeatureTogglesContract.Presenter by inject()
 
-    private val binding: FragmentDebugSettingsBinding by viewBinding()
+    private val binding: FragmentFeatureTogglesBinding by viewBinding()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/debug/featuretoggles/FeatureTogglesFragment.kt:26]: the fix

**Review by kabramovp2p — APPROVED** [review]

File: app/src/main/res/layout/fragment_feature_toggles.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
         app:titleTextAppearance="@style/WalletTheme.TextAppearance.SemiBold18" />
 
     <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/recyclerView"
+        android:id="@+id/rvFeatureToggles"
```
Comment (1sakov) [inline app/src/main/res/layout/fragment_feature_toggles.xml:22]: no way, no rv please

**Review by 1sakov — APPROVED** [review]

File: app/src/main/res/layout/fragment_feature_toggles.xml (hunk 2 above)
Comment (gslevinkov) [reply app/src/main/res/layout/fragment_feature_toggles.xml:22]: recyclerViewFeatureToggles is ok?

Comment (1sakov) [reply app/src/main/res/layout/fragment_feature_toggles.xml:22]: featureTogglesRecyclerView

**Pull request merged**